src/GeositeFramework/js/App.js

```diff
@@ -92,6 +93,14 @@
             // # character before entering hashmodels
             handleHashChangedFn(location.hash === '#' ? '' : location.hash);
             location.hash = "";
+        },
+
+        enableIdentify: function() {
```

Comment:
I renamed these functions to `suspendIdentify` and `resumeIdentify`. I also moved the "identify disable" logic to a better place.